en/contributing/README.md

```diff
@@ -1,16 +1,48 @@
-# Contribute to JabRef
+# Contribute to JabRef [![Join the chat at https://gitter.im/JabRef/jabref](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/JabRef/jabref)
 
 We are really happy that you are interested in contributing to JabRef. Please take your time to look around here. We especially invite you to look into our [community members page](https://discourse.jabref.org/t/community-members/1868?u=koppor) where members introduce themselves.
 
-**I would like to improve the help page. What are the steps?** [![Join the chat at https://gitter.im/JabRef/jabref](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/JabRef/jabref)
+**I would like to try out a feature introduced at pull request. What are the steps?**
 
-Please use the "Edit on GitHub" link in the upper right corner. Then, follow [GitHub's guide to edit files in other user's repository](https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files#editing-files-in-another-users-repository).
+In JabRef, there are dozens of bug fixes and new features introduced using GitHub's pull request mechansim.
+You can browse all at <https://github.com/JabRef/jabref/pulls>.
+The JabRef team really welcomes users to try out these additions and to comment on them.
+Getting in improvements at this stage is much easier than fixing afterwards.
```

Comment:
But last time I wanted to say something in an issue, you said it's feature creep :(

Reply:
I think, its a case-by-case basis 😅. We as team need to see whether its a quick win on an existing issue (e.g., renaming butotns) or a large improvement. If it is an improvment, we need to judge if tis doable in the scope of the PR or should be done as follow-up. - Currently, the priority is on getting PRs merged, as we have more than 10 opened pul requests. - BTW, I am sure, I have not used the word "feature creep" 😇